optimize_workspace.sh

```diff
@@ -1,19 +1,42 @@
 #!/bin/bash
 set -o errexit -o nounset -o pipefail
-command -v shellcheck > /dev/null && shellcheck "$0"
+command -v shellcheck >/dev/null && shellcheck "$0"
 
 export PATH="$PATH:/root/.cargo/bin"
 
 rustup toolchain list
 cargo --version
 
-optimize_workspace.py
+# Delete already built artifacts
+rm -f target/wasm32-unknown-unknown/release/*/*.wasm
 
-# Postprocess artifacts
+# Build artifacts
+echo -n "Building artifacts in workspace..."
+/usr/local/bin/build_workspace.py
+echo "done."
+
+echo -n "Optimizing artifacts in workspace..."
+mkdir -p artifacts
+TMPDIR=$(mktemp -d artifacts.XXX)
+# Optimize artifacts
+(
+  cd "$TMPDIR"
+
+  for WASM in ../target/wasm32-unknown-unknown/release/*/*.wasm
```

Comment:
When we are in $TMPDIR (which could be anywhere on the system), we should not rely in the relative path anymore

Reply:
No, because that is defined / fixed by the `mktemp` template above.

Reply:
A way to make this more portable would be to do something like `DIR=$(mktemp -p $(pwd) artifacts.XXX)`.
I'll do it.

Reply:
I see now that you were probably thinking it was the system `TMPDIR` var. I renamed it for clarity, and to avoid potential issues.

Reply:
Ah, my misunderstanding was that the template argument also desines the parent directory, not only the pattern of the folder name. `mktemp -d artifacts.XXX` was correct then because it means `mktemp -d ./artifacts.XXX`.